### PR TITLE
Filter branch- filters in place

### PR DESCRIPTION
**Pull request by lape15**
I have implemented some of the filters, please review.

### CODE REVIEW

**Comment by Nkemjiks** [comment]
The brand and product type filter default value should be empty

File: src/Components/Pages/Filters.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    <div className={`filter ${filterToggle ? "filter extra" : "filter"} `}>
+      <div className="filter-btn" onClick={handleFilter}>
+        {" "}
+        <i className="fas fa-filter"></i>Filters
+      </div>
+      <div className={`hide ${filterToggle ? " filter-item" : "hide"}`}>
+        <form onSubmit={handleFormSubmit}>
+          <label>
+            Brand:
+            {/*   <input
```
Comment (Nkemjiks) [inline src/Components/Pages/Filters.js:22]: commented code

File: src/Components/Pages/fitlerType.js [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+];
+const product_type = [
+  "pencil",
+  "lipstick",
+  "liquid",
+  "powder",
+  "lip_gloss",
+  "gel",
+  "cream",
+  "",
```
Comment (Nkemjiks) [inline src/Components/Pages/fitlerType.js:68]: Remove this

File: src/Components/Pages/index.js [hunk 3]
```diff
@@ -1,15 +1,105 @@
...
+import axios from "axios";
+import Products from "./Products";
+import Filters from "./Filters";
+import Loading from "../Loader/index";
+
+const URL = "http://makeup-api.herokuapp.com/api/v1/products.json";
+const Homepage = () => {
+  const [products, setProducts] = useState(null);
+  const [filter, setFilter] = useState({
+    brand: "colourpop",
```
Comment (Nkemjiks) [inline src/Components/Pages/index.js:11]: The default value here should be an empty string

File: src/Components/Pages/index.js [hunk 4]
```diff
@@ -1,15 +1,105 @@
...
+    if (filter.brand.length > 0 && filter.productType.length === 0) {
+      try {
+        brandData = await axios.get(
+          `http://makeup-api.herokuapp.com/api/v1/products.json?brand=${filter.brand}`
+        );
+        if (brandData.data) {
+          setProducts(brandData.data);
+        }
+      } catch (error) {
+        console.log(error);
```
Comment (Nkemjiks) [inline src/Components/Pages/index.js:53]: Don't you think your user's will want to see this error?

File: src/Components/Pages/index.js [hunk 5]
```diff
@@ -1,15 +1,105 @@
...
+    if (filter.brand.length > 0 && filter.productType.length > 0) {
+      try {
+        brandData = await axios.get(
+          `http://makeup-api.herokuapp.com/api/v1/products.json?brand=${filter.brand}&product_type=${filter.productType}`
+        );
+        if (brandData.data) {
+          setProducts(brandData.data);
+        }
+      } catch (error) {
+        console.log(error);
```
Comment (Nkemjiks) [inline src/Components/Pages/index.js:66]: Same as here

File: src/Components/Pages/index.js (hunk 4 above)
Comment (lape15) [reply src/Components/Pages/index.js:68]: So i should show it?

**Pull request merged**